src/objc/properties.rs

```diff
@@ -62,3 +63,32 @@ pub(super) fn objc_setProperty(
         release(env, old);
     }
 }
+
+// note: https://opensource.apple.com/source/objc4/objc4-723/runtime/objc-accessors.mm.auto.html
+//       says that hasStrong is unused, but other versions of the source seem to use it. Not sure what to do here.
+pub(super) fn objc_copyStruct(
+    env: &mut Environment,
+    dest: MutVoidPtr,
+    src: ConstVoidPtr,
+    size: GuestUSize,
+    _atomic: bool,
+    _hasStrong: bool,
+) {
+    // TODO: implement atomic and hasStrong support
```

Comment:
The source code you reference suggests `_hasStrong` is always unused, so maybe we can ignore it?

As for the atomic, I think it would be good to `assert!(!atomic);` so we'll catch it if an app needs it.